City-of-Helsinki/drupal-helfi-platform-config
# UHF-2093 Update helper

How to test on a new setup:
- `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-UHF-2093_update_helper hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
- `cd hel-platform`
- Switch to corresponding branches: 
  - `composer require drupal/helfi_platform_config:dev-UHF-2093_update_helper`
- Run `make new`
  - Check that the new site installs without fatal errors
  - Click on the login link and check that the site has been installed as it normally would

For other instances, check instructions from their repositories.
Sote: https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/67
Kymp: https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/64
SJT: https://github.com/City-of-Helsinki/drupal-helfi-strategia/pull/14
Helfi: https://github.com/City-of-Helsinki/drupal-helfi/pull/180
  
Also check: https://github.com/City-of-Helsinki/drupal-helfi-platform/pull/45
